Core/Inc/config.h

```diff
@@ -31,11 +38,17 @@
 
 // Helper Macros
 
+#ifdef LWIP_HDR_DEF_H // def.h
```

Comment:
should the define for this also be located in this file?

Reply:
This is just for the ethernet stuff; the ```#ifdef``` makes sure that the macro and static function only exist in files that need it.